server/apparmor/apparmor_supported.go

```diff
@@ -47,13 +36,11 @@ type profileData struct {
 	Version int
 }
 
-// InstallDefaultAppArmorProfile installs default apparmor profile.
-func InstallDefaultAppArmorProfile() {
-	if err := InstallDefault(defaultApparmorProfile); err != nil {
-		// Allow daemon to run if loading failed, but are active
-		// (possibly through another run, manually, or via system startup)
-		if !IsLoaded(defaultApparmorProfile) {
-			logrus.Errorf("AppArmor enabled on system but the %s profile could not be loaded.", defaultApparmorProfile)
+// LoadDefaultAppArmorProfile loads default apparmor profile, if it is not loaded.
+func LoadDefaultAppArmorProfile() {
+	if !IsLoaded(DefaultApparmorProfile) {
```

Comment:
1. will fix
2. will fix
3. this is required by @runcom in [this pr](https://github.com/kubernetes-incubator/cri-o/pull/215#discussion_r90755948) , so what do you think about this? @runcom 
4. The configured profile should be installed by user, and user passes the name through flag. [This line](https://github.com/kubernetes-incubator/cri-o/blob/master/server/server.go#L305) will store the name in struct `Server`.